tests/scripts/thread-cert/pktverify/layer_fields.py

```diff
@@ -275,7 +275,7 @@ def __call__(self, v: Union[LayerFieldsContainer, LayerField]):
     'mle.tlv.route64.nbr_out': _list(_auto),
     'mle.tlv.route64.nbr_in': _list(_auto),
     'mle.tlv.route64.id_seq': _auto,
-    'mle.tlv.route64.id_mask': _auto,
+    'mle.tlv.route64.id_mask': _str,
```

Comment:
```suggestion
    'mle.tlv.route64.id_mask': _routerid_set,
```

nit: Use a method to convert `mle.tlv.route64.id_mask` to a set directly?

The _routerid_set can convert the field value to a set. 

```
def _routerid_set(v: Union[LayerFieldsContainer, LayerField]) -> set:
   ...
    
```